src/app/components/Header/index.js

```diff
@@ -6,13 +6,17 @@ import style from './Header.css'
 
 export default class Header extends Component {
   render() {
+    const { noMenu } = this.props
```

Comment:
This code might more clear if it was written in a more declarative way. Something like: `const { hasMenu = false } = this.props`. This way, we can be more truthy, and we don't need to add configuration to take away a thing we don't need, we only have to add add configuration for a thing we want.

Reply:
Good call. I'll update it.